internal/cmd/asyncapi/command_export.go

```diff
@@ -174,6 +172,7 @@ func (c *command) getChannelDetails(details *accountDetails, flags *flags) error
 		log.CliLogger.Warnf("Failed to get tags: %v", err)
 	}
 	details.channelDetails.example = nil
+	var err error
 	if flags.consumeExamples {
 		details.channelDetails.example, err = c.getMessageExamples(details.consumer, details.channelDetails.currentTopic.GetTopicName(), details.channelDetails.contentType, details.srClient, flags.valueFormat)
```

Comment:
The more idiomatic approach for cases like this one is to do `example, err :=` which lets us use `:=` to shorten the scope of `err`, followed by an assignment, i.e. `details.channelDetails.example = example`.